### PR TITLE
Add color server nodelet support

### PR DESCRIPTION
**Pull request by clunietp**
Added support for running color server as a nodelet.  Tested on melodic

### CODE REVIEW

File: octomap_server/CMakeLists.txt [hunk 1]
```diff
@@ -73,6 +73,9 @@ target_link_libraries(octomap_tracking_server_node ${PROJECT_NAME} ${LINK_LIBS})
 add_library(octomap_server_nodelet src/octomap_server_nodelet.cpp)
 target_link_libraries(octomap_server_nodelet ${PROJECT_NAME} ${LINK_LIBS})
 
+add_library(octomap_color_server_nodelet src/octomap_server_nodelet.cpp)
```
Comment (wxmerkt) [inline octomap_server/CMakeLists.txt:76]: Thank you for your contribution. Can you please also add this nodelet to the `install` command below?

Thank you!

**Review by wxmerkt — CHANGES_REQUESTED** [review]

**Comment by wxmerkt** [comment]
Thank you for your contribution!

**Comment by clunietp** [comment]
Thank you @wxmerkt !

**Comment by clunietp** [comment]
@wxmerkt , I just noticed this in `octomap_mapping/CMakeLists.txt`:

    install(FILES nodelet_plugins.xml
    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})

I'm assuming we'd want to add `color_nodelet_plugins.xml` to this as well?

**Comment by wxmerkt** [comment]
Yes, please. Thanks for spotting!

**Pull request merged**